llama_index/embeddings/bedrock.py

```diff
@@ -0,0 +1,184 @@
+from typing import Any, List, Optional
+import warnings
+import json
+import os
+from llama_index.bridge.pydantic import PrivateAttr
+from llama_index.embeddings.base import BaseEmbedding
+
+Embedding = List[float]
+
+
+class Models(str):
+    TITAN_EMBEDDING = "amazon.titan-embed-text-v1"
+
+
+# To be used when AWS Bedrock comes with more embedding models
+# class ModelParams(str):
+# TITAN_EMBEDDING = {"inputText":""}
+
+
+class BedrockEmbeddings(BaseEmbedding):
+    _client: Any = PrivateAttr()
+
+    def __init__(
+        self,
+        model_name: str = Models.TITAN_EMBEDDING,
+        max_length: int = 8000,
```

Comment:
I don't see this being used anywhere?

Reply:
Currently max_length is not being used as the query is truncated automatically once the length crosses it's limit (i.e. 8000). I'll set a warning for query crossing the limit right away.

Reply:
I wanted to set warning to check if the number of tokens exceed the limit, however boto3 handles this, hence I am removing it.

Reply:
Hi @Disiok , I have made the changes, please review.